pkgs/top-level/python-packages.nix

```diff
@@ -21672,8 +21682,11 @@ in {
       export LANG="en_US.UTF-8";
     '';
 
-    patchPhase = ''
-      sed -i "s|_lgeos = load_dll('geos_c', fallbacks=.*)|_lgeos = load_dll('geos_c', fallbacks=['${pkgs.geos}/lib/libgeos_c.so'])|" shapely/geos.py
+    patchPhase = let
+      libc = if stdenv.isDarwin then "libc.dylib" else "libc.so.6";
+    in ''
+      sed -i "s|_lgeos = load_dll('geos_c', fallbacks=.*)|_lgeos = load_dll('geos_c', fallbacks=['${pkgs.geos}/lib/libgeos_c.${sharedLibraryExtension}'])|" shapely/geos.py
```

Comment:
Probably a dot too much: '/nix/store/k15091bkbbvfhjjcmi7pggs4c212gkfh-geos-3.6.1/lib/libgeos_c..so'

```diff
- sed -i "s|_lgeos = load_dll('geos_c', fallbacks=.*)|_lgeos = load_dll('geos_c', fallbacks=['${pkgs.geos}/lib/libgeos_c.${sharedLibraryExtension}'])|" shapely/geos.py
+ sed -i "s|_lgeos = load_dll('geos_c', fallbacks=.*)|_lgeos = load_dll('geos_c', fallbacks=['${pkgs.geos}/lib/libgeos_c${sharedLibraryExtension}'])|" shapely/geos.py
```

Reply:
is this different on OS X?

Reply:
Currently the build is still breaking on Linux.